library/core/src/ffi/primitives.rs

```diff
@@ -0,0 +1,163 @@
+//! This module provides primitive types which will
+//! match those defined by C, so that code that interacts with C will
+//! refer to the correct types.
+//!
+//! primitives.rs only contains type definitions for consistent external parsing
```

Comment:
Doc comment style nit

```suggestion
//! `primitives.rs` only contains type definitions for consistent external parsing.
```

Reply:
Honestly I am just going to take this line out, don't think it really provides any valuable context. 

I went with this as the only doc comment at the top: 

```
//! Provides C-compatible primitive types to ensure correct type matching when interfacing with C code.
```

Reply:
It's fine to remove this line but please do mention somewhere that it needs to be kept usable as a standalone module. Just to avoid confusion when somebody down the line adds a `use super::something` to this module and gets confused when things break 🙂